### PR TITLE
fix(paginator): update the shape of the paginator

### PR DESCRIPTION
**Pull request by Gio85**

### CODE REVIEW

File: src/types.ts [hunk 1]
```diff
@@ -128,11 +132,11 @@ export enum DEFAULT_PAGINATOR {
 }
 
 export interface IPaginator {
-    from: number
-    size: number
+    from?: number
```
Comment (useless-stuff) [inline src/types.ts:135]: can u use partial inside the methods and keep these interfaces ( also IFindDocumentsInput ) as they really are?

Comment (useless-stuff) [reply src/types.ts:135]: having interfaces with all question marks brakes the logic of the interface it self

**Review by julian-aimusic — APPROVED** [review]

File: src/types.ts (hunk 1 above)
Comment (Gio85) [reply src/types.ts:135]: sure

**Review by julian-aimusic — APPROVED** [review]

**Review by useless-stuff — APPROVED** [review]

**Pull request merged**